modules/gapi/include/opencv2/gapi/infer/ov.hpp

```diff
@@ -702,6 +725,14 @@ namespace detail
     {
         static const char* tag() { return "gapi.wip.ov.benchmark_mode"; }
     };
+    template<> struct CompileArgTag<cv::gapi::wip::ov::workload_type>
```

Comment:
Just note: _I think we will need the same compile argument ONNXRT backends after._